src/store/actions/payment.js

```diff
@@ -516,24 +517,31 @@ export const putBalanceProof = (message, payment) => async (
   }
 };
 
-export const putNonClosingBalanceProof = (message, payment) => async (
-  dispatch,
-  getState,
-  lh
-) => {
+export const putNonClosingBalanceProof = (
+  message,
+  payment,
+  dataToPack = null
+) => async (dispatch, getState, lh) => {
   const { getAddress } = ethers.utils;
-  const dataToSign = getDataToSignForNonClosingBalanceProof(message);
+  const dataForPack = dataToPack || message;
+  const isLT = dataToPack !== null;
+  const dataToSign = isLT
```

Comment:
I dont think this is ok.

Basically you invoke one method when is a locked transfer, and you invoke another when is NOT a lockedtransfer, but.. you specifiy on that method the isLT boolean, but you already know that is not a locked transfer

Reply:
I changed it, check again now